doc/modules/clustering.rst

```diff
@@ -528,24 +549,25 @@ computed using a function of a gradient of the image.
 
 .. topic:: Examples:
 
- * :ref:`sphx_glr_auto_examples_cluster_plot_segmentation_toy.py`: Segmenting objects
-   from a noisy background using spectral clustering.
+  * :ref:`sphx_glr_auto_examples_cluster_plot_segmentation_toy.py`: Segmenting
+    objects from a noisy background using spectral clustering.
 
- * :ref:`sphx_glr_auto_examples_cluster_plot_coin_segmentation.py`: Spectral clustering
-   to split the image of coins in regions.
+  * :ref:`sphx_glr_auto_examples_cluster_plot_coin_segmentation.py`: Spectral
+    clustering to split the image of coins in regions.
 
-.. |coin_kmeans| image:: ../auto_examples/cluster/images/sphx_glr_plot_coin_segmentation_001.png
+  .. |coin_kmeans| image:: ../auto_examples/cluster/images/sphx_glr_plot_coin_segmentation_001.png
```

Comment:
This does not need to be indented I think, and same for the two other images below.